pkgs/top-level/all-packages.nix

```diff
@@ -1546,8 +1546,16 @@ with pkgs;
 
   fxlinuxprintutil = callPackage ../tools/misc/fxlinuxprintutil { };
 
+  gay = callPackage ../tools/misc/gay {
+    inherit (python3Packages) buildPythonPackage fetchPypi;
+  };
+
   genann = callPackage ../development/libraries/genann { };
 
+  gay = callPackage ../tools/misc/gay {
+    inherit (python3Packages) buildPythonPackage fetchPypi;
+  };
```

Comment:
This is duplicated and also it is easier to use python3Packages.buildPythonPackage in the file itself.